### DISCUSSION TITLE
About Orbit Documentation

### DISCUSSION THREAD

**Discussion by tinchox5** [zumerlab/orbit#17] [Polls]
Help us to understand how users perceive the quality of our documentation.

**Comment by SanderElias** [comment]
The documentation is crisp and clear.
What I found missing is the sources of the examples.

**Reply by tinchox5** [reply]
Thank you for your feedback @SanderElias !!
You are right I'm going to add source code. 

Which do you prefer a codepen or the code below each example?

**Reply by SanderElias** [reply]
Is both an option?

I would like to see the code in the same way as in the rest of the docs, 
But the examples would be nice to have in a code-pen, so you can play around with them a bit before "committing"

**Reply by tinchox5** [reply]
Perfect! Yes, no problem. I'll improve example documentation with your suggestions. I hope next week

**Reply by tinchox5** [reply]
All set (souce code + codepen)! 

**Docs**: https://zumerlab.github.io/orbit-docs/examples/

I made a Orbit Codepen Collection at https://codepen.io/collection/KwVyZM

**Note**: I removed  solar system example because I don't like very much, but its codepen is https://codepen.io/zircle/pen/LYKggZy

**Reply by SanderElias** [reply]
@tinchox5 
Thanks, that makes getting started a load easier!

**Reply by tinchox5** [reply]
Good! Any doubts just tell me. Glad to help
